docs/eigenlayer/rewards-claiming/rewards-claiming-overview.md

```diff
@@ -9,27 +9,43 @@ title: Rewards Claiming Overview
 
 The EigenLayer Rewards protocol enables AVSs to make rewards to stakers and operators. Operators earn rewards by opting in to AVSs that make `RewardsSubmissions` to the `RewardsCoordinator`, a core protocol contract. Within a single `RewardsSubmission`, an AVS can specify a time range for which the reward will be distributed, a list of weights for each `Strategy` for the reward, and an ERC20 token to make rewards in.  
 
-Operators will earn a flat 10% commission on rewards. The rest of the reward is passed on to the operator's delegated stakers. Rewards are proportional to:
+By default, Operators will earn a flat 10% commission on rewards. The rest of the reward is passed on to the operator's delegated Restakers. Rewards are proportional to:
 - The amount of stake.
 - The AVS's relative weighting of strategies in a rewards submission.
 
-Rewards are calculated via an offchain process. Every week a merkle root is posted which represents the cumulative rewards across all earners. There is an additional 2 hour delay on testnet and 1 week delay on mainnet after posting in order for the root to be claimable against with a valid merkle proof. The deterministic calculation of the distribution of rewards is specified in our [technical docs](https://github.com/Layr-Labs/eigenlayer-contracts/blob/dev/docs/core/RewardsCoordinator.md). 
 
-Reward Earners (Stakers and Operators) can set a claimer address that can claim rewards for the tokens they've earned. An Earner is its own claimer by default and only the claimer address can claim rewards. If a new claimer is set, the new address can claim all of the previously unclaimed rewards. The earner can always configure their designated claimer address.
+## Rewards v2 (currently in Tesnet)
+
+Rewards v2 addresses key challenges identified in the EigenLayer Rewards MVP (v1), particularly the need for operator-directed rewards via external logic and flexibility for operators in setting different fee rates to better cover operating costs or attract more stake when running different AVS types. Rewards v2 adds the following use cases:
+- Operator-directed rewards from AVSs, allowing AVSs to segment and reward specific operators. This gives AVSs the flexibility to set custom logic for rewards to individual operators, based on work completed or anything else they may design or desire (e.g., more equal distribution of operator support for decentralization or security reasons).
```

Comment:
todo, add explanation for Variable Operator Fees  and Operator Splits for Programmatic Incentives